models/paraswap/zkevm/paraswap_v6_zkevm_trades.sql

```diff
@@ -0,0 +1,135 @@
+{{ config(
+    schema = 'paraswap_v6_zkevm',
+    alias = 'trades',    
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'method', 'trace_address'],
+    post_hook='{{ expose_spells(\'["zkevm"]\',
+                                "project",
+                                "paraswap_v6",
+                                \'["eptighte", "mwamedacen"]\') }}'
```

Comment:
We can apply the new format for post_hook here:
```suggestion
    post_hook='{{ expose_spells(blockchains = \'["zkevm"]\',
                                spell_type = "project",
                                spell_name = "paraswap_v6",
                                contributors = \'["eptighte", "mwamedacen"]\') }}'
```